scripts/_helpers.py

```diff
@@ -551,7 +551,7 @@ def country_name_2_two_digits(country_name):
         2-digit country name
     """
     if (
-        country_name
+        country_name.any()
```

Comment:
This is no the intended behavior. It should be removed

Reply:
Okay! So, your answer to my second questions (attached to the review notification) is "no". Agree, the `where` modification doesn't look great. Commit reverted